File: task-03/Javascript/problem.js
function is_prime(n){
    if (n<2){
        return false;
    }

    for (let i =2; i<n; i++){
        if (n%i===0){
            return false;
        }
    }
return true;

}
const readline = require('readline').createInterface({
    input:process.stdin,
    output: process .stdout
});
 readline.question("Enter a number:", input => {
    const N = parseInt(input);

    console.log(`Prime number up to ${N} are:`);

    for (let i=2; i <=N; i++){
        if (is_prime(i)){
            process.stdout.write(i+ ' ');

        }
    }
    readline.close();
 });

File: task-03/Go/problem.go
package main

import "fmt"

func is_prime(n int) bool {
	if n < 2 {
		return false
	}
	for i := 2; i < n; i++ {
		if n%i == 0 {
			return false
		}
	}

	return true
}

func main() {
	var N int
	fmt.Println("Enter a number")
	fmt.Scanln(&N)

	for i := 2; i <= N; i++ {
		if is_prime(i) {
			fmt.Print(i, " ")
		}
	} 
	fmt.Println("\n")
}